.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,68 @@
+name: Nightly Release
+
+on:
+  workflow_dispatch:  # Manual trigger
+  schedule:
+  - cron: '0 5 * * *' # 5 AM UTC = Midnight EST
+
+jobs:
+  nightly:
+    if: ${{ github.repository == 'shipwright-io/triggers' }}
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write  # To be able to get OIDC ID token to sign images.
+      contents: write  # To be able to update releases.
+      packages: write  # To be able to push images and signatures.
+
+    env:
+      IMAGE_HOST: ghcr.io
+      IMAGE_NAMESPACE: ${{ github.repository }}
+
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v3
+      with:
+        go-version: '1.19.x'
+        cache: true
+        check-latest: true
+
+    # Install tools
+    - uses: ko-build/setup-ko@v0.6
+      with:
+        version: v0.13.0
+    - uses: imjasonh/setup-crane@e82f1b9a8007d399333baba4d75915558e9fb6a4
+    - uses: sigstore/cosign-installer@v3
+
+
+    - name: Get current date
+      id: date
+      run: echo "date=$(date +'%Y-%m-%d-%s')" >> $GITHUB_OUTPUT
+
+    - name: Install helm
+      run: |
+          curl -LO https://get.helm.sh/helm-v3.8.0-linux-amd64.tar.gz
+          tar -zxvf helm-v3.8.0-linux-amd64.tar.gz
+          mv linux-amd64/helm /usr/local/bin/helm
+          helm version
```

Comment:
Please make use of the [azure/setup-helm](https://github.com/Azure/setup-helm) action.

Reply:
fixed